app/templates/partials/summary/display-question.html

```diff
@@ -1 +1 @@
-<div class="summary__question u-pb-s" id="{{ question.id }}" data-qa="summary-question-title">{{ question.number or ''}} {{question.title|striptags }}</div>
+<div class="summary__question" id="{{ question.id }}" data-qa="summary-question-title">{{ question.number or ''}} {{question.title|striptags }}</div>
```

Comment:
This template is used for questions in general. So is it intended that there should be no `padding-bottom` in all cases or just the hours case?

Expected?
Before:
<img width="598" alt="screenshot 2018-12-05 at 11 09 04" src="https://user-images.githubusercontent.com/35296336/49509789-490da980-f87e-11e8-8dd5-84e2f6440269.png">
After:
<img width="600" alt="screenshot 2018-12-05 at 11 09 16" src="https://user-images.githubusercontent.com/35296336/49509790-490da980-f87e-11e8-83fe-2eb526c6c118.png">

Reply:
It should be in all cases, I think the total was the first place it was noticed.